examples/deepzoom/static/openseadragon-scalebar.js

```diff
@@ -1,4 +1,4 @@
-/* 
+/*
```

Comment:
Let's just ignorelist vendored sources.  These sorts of changes make it harder to compare the vendored sources to external copies, and make update diffs harder to read.